### ISSUE TITLE
NPE Crash when setting regulator value in puller(?) module

### ISSUE THREAD

**Issue by shabble** [desht/ModularRouters#12]
Created I think a puller mk1 (`modularrouters:module:4` if that helps) with regulator addition.

* Accessed module GUI from hotbar with r-click to open.
* Noticed tooltip over regulator region mentioning scrollwheel support
* Attempted to scroll with cursor over text/numeral field
* Crash

crashlog: https://gist.github.com/shabble/ae65fb1e53f99154e73fbbd9cb1dfe9b

```
java.lang.NullPointerException: Updating screen events
at me.desht.modularrouters.gui.module.GuiModule.func_146284_a(GuiModule.java:174)
...
```

* MC 1.10.2
* modular-routers-1.10.2-1.2.5.jar

Running a lightly bastardised SkyFactory 3 modpack otherwise.

Haven't tried to see if reproduceable, can check if useful.

**Comment by desht** [comment]
Yeah, I think I see the problem - should be an easy fix, but not on my dev machine right now.

**Comment by desht** [comment]
OK, so the crash happens when left-clicking the the little regulator icon (where the tooltip pops up).  That's a simple bug which is an easy fix.

I couldn't get any crash when scrolling either with mouse or keyboard over the numeric textfield - that all works fine.  Can you just quickly re-check so I'm not missing anything?  Thanks.

**Comment by shabble** [comment]
sure - might have mistakenly fat-fingered my trackpad and clicked instead. Will test later this evening

**Comment by shabble** [comment]
Yep, it's clicking the icon that does it. Wheeling works fine, as does clicking in the input field itself.

**Comment by desht** [comment]
Ok, that's good then.  Will be fixed in the next release (soonish).  Meantime avoid left-clicking the icon :)

**Comment by desht** [comment]
Fixed in 1.3.0/2.1.0 releases, just uploaded.
